docs/dialogue-elements/small-talk.rst

```diff
@@ -107,7 +107,7 @@ First, you need NLU data for reactions and acknowledgements:
 
 And then you need training stories to teach Rasa how to respond:
 
-.. code-block:: story
+.. code-block:: md
```

Comment:
why shouldn't these be stories?

Reply:
"story" is not a proper lexer format. Perhaps you can define your own lexer format -- if so, the definition got lost somewhere, because Sphinx did not recognize it

Reply:
ah I see. the code to add the lexer is here: https://github.com/RasaHQ/rasa_core/blob/master/docs/core/conf.py#L383

the code for the lexer is this folder https://github.com/RasaHQ/rasa_core/tree/master/docs/core/util

Reply:
awesome thank you 👍